src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -762,13 +762,19 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
 
   private addAllCategoriesButton() {
     const allCategories = this.categoryFacetTemplates.buildAllCategoriesButton();
-    allCategories.on('click', () => {
-      this.reset();
-      this.scrollToTop();
-    });
+    new AccessibleButton()
+      .withElement(allCategories)
+      .withClickAction(() => this.goBackToAllcategories())
+      .withEnterKeyboardAction(() => this.goBackToAllcategories())
```

Comment:
`withSelectAction` takes care of both click and enter 😄

Reply:
Thanks! Replacing that.